docs/sources/installation/_index.md

```diff
@@ -8,11 +8,11 @@ weight: 200
 
 Instructions for different methods of installing Loki and Promtail.
 
-- [Install using Tanka (recommended)](tanka/)
-- [Install through Helm](helm/)
-- [Install through Docker or Docker Compose](docker/)
-- [Install and run locally](local/)
-- [Install from source](install-from-source/)
+- [Install using Tanka (recommended)](tanka.md/)
```

Comment:
I don't think the "/" needs to stay. Same for the lines below.
Can you please remove them as well while you are at it?